python/dllib/src/bigdl/dllib/utils/common.py

```diff
@@ -628,8 +629,12 @@ def callBigDlFunc(bigdl_type, name, *args):
             result = callJavaFunc(api, *args)
         except Exception as e:
             error = e
-            if "does not exist" not in str(e):
+            regex = re.compile(r'Method\s' + name + r'.*does not exist')
+            if regex.match(str(e)):
                 invalidOperationError(False, str(e), cause=e)
```

Comment:
if regex.match, which means the error is method not exist, we should skip this error by doing nothing, not call invalidOperationError

Reply:
updated